modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/DataGenerationApplication.java

```diff
@@ -34,13 +35,13 @@ public DataGenerationApplication(Ignite ignite) {
     }
 
     /** {@inheritDoc} */
-    @Override protected void run(String[] args) {
+    @Override protected void run(Map<String, String> args) {
```

Comment:
Why do you use Map instead of fair JsonNode object? Now you're limited with flat structure and user responsible to cast value types.

Reply:
+1 JsonObject seems to be more universal solution and has a potential to extend

Reply:
Done